core/domain/feedback_domain.py

```diff
@@ -22,7 +22,7 @@ class FeedbackThread(object):
     """Domain object for a feedback thread.
 
     Attributes:
-        full_thread_id: str. The feedback thread ID.
+        thread_id: str. The feedback thread ID.
         exploration_id: str. The associated exploration's ID.
```

Comment:
Hm, how are you planning to generalize this for other types of threads?

Reply:
The generalization plan: 
- Thread_id becomes entity_type.entity_id.<random str> 
- exploration_id -> entity_id
- state_name should become a more general parameter. Probably along the lines of customization_args or thread_params or similar.

Reply:
OK. I guess that's a separate PR? In that case please add a TODO here and ideally an issue to track it as well, since this is a codebase deficiency until it gets addressed (and not merely a new feature). Please link to the issue in this comment once you've done so.

Reply:
Yes. Actually it's a task in my second milestone. I'll add an issue to track it as well.

Reply:
Added issue #4995 with all follow up steps.

Reply:
Added TODO